### PR TITLE
[DOCS-6566] - Add logs

### PR DESCRIPTION
**Pull request by oliveralfresco2018**

### CODE REVIEW

**Review by andrea-ligios — COMMENTED** [review]
You might want to add that 
- the Runtime Bundle Logs and the Storage Logs belongs to the app and are unavailable if the app is not deployed correctly 
- the Deployment Service Logs belongs to a shared service, are always available, and do highlight (with a different blue background colour) the lines related to that app (whose three dots' context menu has been used to request the logs)

**Review by andrea-ligios — APPROVED** [review]
LGTM!

File: process-automation/latest/admin/release.md [hunk 1]
```diff
@@ -199,3 +199,19 @@ To update the runtime version.
...
+
+### Logs
+
+You can view Deployment Service Logs, Runtime Bundle Logs, and Process Storage Logs to help understand any errors you may be having with your installation. The Deployment Service Logs are always available because they belong to a shared service. The logs use different blue highlighting to indicate the app being viewed. The Runtime Bundle Logs and the Storage Logs are unavailable if the app is not deployed correctly.
+
+To view the logs.
+
+1. Log into the Alfresco Admin App.
+
+2. Expand **Devops** from the left pane and select **Application Instances**.
```
Comment (anxumalo) [inline process-automation/latest/admin/release.md:211]:
```suggestion
2. Expand **DevOps** from the left pane and select **Application Instances**.
```

File: process-automation/latest/admin/release.md [hunk 2]
```diff
@@ -199,3 +199,19 @@ To update the runtime version.
 > **Note:** You can also update the runtime version of an application when you upgrade. For more see [Upgrade](#upgrade)
 
 ![update runtime]({% link process-automation/images/update-runtime.png %})
+
+### Logs
+
+You can view Deployment Service Logs, Runtime Bundle Logs, and Process Storage Logs to help understand any errors you may be having with your installation. The Deployment Service Logs are always available because they belong to a shared service. The logs use different blue highlighting to indicate the app being viewed. The Runtime Bundle Logs and the Storage Logs are unavailable if the app is not deployed correctly.
+
+To view the logs.
```
Comment (anxumalo) [inline process-automation/latest/admin/release.md:207]:
```suggestion
To view the logs:
```

File: process-automation/latest/admin/release.md [hunk 3]
```diff
@@ -199,3 +199,19 @@ To update the runtime version.
 > **Note:** You can also update the runtime version of an application when you upgrade. For more see [Upgrade](#upgrade)
```
Comment (anxumalo) [inline process-automation/latest/admin/release.md:199]:
```suggestion
> **Note:** You can also update the runtime version of an application when you upgrade. For more see [Upgrade](#upgrade).
```

**Review by gravitonian — APPROVED** [review]
Built and checked the page with change. Looks good.

File: process-automation/latest/admin/release.md (hunk 1 above)
Comment (oliveralfresco2018) [reply process-automation/latest/admin/release.md:211]: its not written like that! cheers :-)

Comment (anxumalo) [reply process-automation/latest/admin/release.md:211]: Ok - I was following what's at the top of the page.

**Review by anxumalo — APPROVED** [review]

**Pull request merged**